src/v2/Components/LocationAutocompleteInput.tsx

```diff
@@ -220,7 +223,7 @@ export const normalizePlace = (place?: Place): Location => {
     stateCode: components.state?.short_name,
     postalCode: components.postalCode?.long_name,
     country: components.country?.long_name,
-    countryCode: components.country?.short_name,
+    countryCode: withCountryCode ? components.country?.short_name : undefined,
```

Comment:
**question (non-blocking):** I'm curious why were are using `undefined` rather than `null`.

My understanding is that it's preferable to not use `undefined` manually and bias to `null` for values that represent an empty value.

Reply:
I used `undefined` here to prevent the `countryCode` field from being sent to the server, if I use `null` here the Relay will think that I want to send the field to the server with `null` value, but if I'll send `undefined` the Relay will ignore the field.

Reply:
That's interesting to know! However, I worry that your intention behind using `undefined` won't be communicated to future developers who encounter this code (and we'll end up with the same bug if they change it). This fix is urgent so I also think that this is (non-blocking) but it would be good to take this object out of the `return` statement and conditionally add `countryCode` to it so that the intention is clearer.

Reply:
Generally I want to back to the previous version of the `normalizePlace` and always return the `countryCode`, but update places where `normalizePlace` uses to use only needed fields. That should save code from the same problem in future(adding new field).

Reply:
Cool. I'm looking forward to checking that out in the follow-up PR 🙌